app/src/main/java/com/example/insights/ItemAdapter.kt

```diff
@@ -12,18 +12,18 @@ import androidx.cardview.widget.CardView
 import androidx.core.content.ContextCompat.startActivity
 import androidx.recyclerview.widget.RecyclerView
 
-class ItemAdapter(val context: BookFragment, val items: ArrayList<HashMap<String, String>>) :
-    RecyclerView.Adapter<ItemAdapter.ViewHolder>() {
+class BookRecylerViewAdapter(val context: BookFragment, val items: ArrayList<HashMap<String, String>>) :
```

Comment:
Sorry @misaka3 , there was a typo in the issue #43 (edited now) , the name for the new class should BookRecyclerViewAdapter and not BookRecylerViewAdapter. Please make the neccessary changes in the two files